.github/workflows/linters.yml

```diff
@@ -18,6 +18,7 @@ jobs:
         run: npm install -g @lhci/cli@0.11.x
       - name: Lighthouse Report
         run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+
```

Comment:
Hello, thank you for submitting your project. I have noticed that you have removed web linters from your linters.yml file. This is a serious violation of the Code review guideline and it is not acceptable. The linters file is provided by Microverse to ensure that your code follows the best practices and standards. Removing web linters can compromise the quality and integrity of your code and your project. Therefore, you should never remove or edit the linters file in any way.

I have decided to invalidate your project because of this violation. You will have to resubmit your project with the original linters file and make sure that your code passes all the linter checks without any errors or warnings with the exception of webhint. This will ensure that your code is clean, consistent and professional.

We are aware that most of you are experiencing Webhint issues, we are currently working on a solution👨‍💻. For now, please go ahead and request for review if you only have the Webhint error on your PR. Our CRs would be happy to review your PRs 🙂.

Please restore the original linters file in your pull request and resubmit your project for another review. I apologize for the inconvenience. 🙏

Reply:
Webhints is not required since we get the announcement. It make trouble now.  I found the below text at slack-announcement :

Marco Lizza
  [12:12 PM](https://microverse-students.slack.com/archives/C47D4N35M/p1686654751539179)
Hey Micronauts @channel! :wave:
I hope you're doing well! We are aware that most of you are experiencing Webhint issues, we are currently working on a solution:male-technologist:.
For now, please go ahead and request for review if you only have the Webhint error on your PR. Our CRs would be happy to review your PRs :slightly_smiling_face:.
Feel free to share this announcement on your PR for the CR to see:bangbang:
Thank you for your understanding.

Reply:
@HossainAraf The message says submit your project with webhint errors, not remove webhint linters entirely.